server/models/document.py

```
@@ -0,0 +1,41 @@
+    
+    def to_dict(self):
+        return {
+            "id": self.id,
+            "admin_id": self.admin_id,
+            "description": self.description,
+            "file_path": self.file_path,
+            "created_at": self.created_at.isoformat() if self.created_at else None,
+        }
+        
+    @validates("admin_id")
+    def validate_admin_id(self, key, admin_id):
```

Ruff (0.12.2)

Unused method argument: `key`

(ARG002)